tests/Jobs/ElasticSearchIndexInitTest.php

```diff
@@ -0,0 +1,115 @@
+<?php
+
+namespace Tests\Jobs;
+
+use App\User;
+use Illuminate\Foundation\Testing\DatabaseTransactions;
+use Tests\TestCase;
+use App\Jobs\ElasticSearchIndexInit;
+use App\Http\Curl\HttpRequest;
+use App\WikiManager;
+use App\WikiSetting;
+use App\Wiki;
+
+class ElasticSearchIndexInitTest extends TestCase
+{
+    use DatabaseTransactions;
+
+    private $wiki;
+    private $user;
+
+    public function setUp(): void {
+        parent::setUp();
+
+        $this->user = User::factory()->create(['verified' => true]);
+        $this->wiki = Wiki::factory()->create();
+        WikiManager::factory()->create(['wiki_id' => $this->wiki->id, 'user_id' => $this->user->id]);
+        WikiSetting::factory()->create(
+            [
+                'wiki_id' => $this->wiki->id,
+                'name' => WikiSetting::wwExtEnableElasticSearch,
+                'value' => false
```

Comment:
This could maybe se set to true, though that would make the assertion further down harder.
The assertion would need to switch to checking if the job was successful instead?

OR keep this as false, but perhaps leave a note about why? (So someone else doesn't set it to true and essentially eliminate the assertion of this test.

Reply:
done